### PR TITLE
Bug 1221655 - Clearing logcat with DELETE /log

### PR DESCRIPTION
**Pull request by eliperelman**

### CODE REVIEW

File: src/fxos/getProperties.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const EOL = require('os').EOL;
```
Comment (gaye) [inline src/fxos/getProperties.js]: https://twitter.com/wycats/status/604491094425018369 :D

File: src/adb/logcat.js [hunk 2]
```diff
@@ -1,3 +1,9 @@
-module.exports = function logcat() {
-  return this.spawn('logcat');
+module.exports = function logcat(timestamp) {
+  let args = ['logcat'];
+
+  if (timestamp) {
+    args.push(...['-T', timestamp]);
```
Comment (gaye) [inline src/adb/logcat.js]:
```js
args.push('-T', timestamp);
```
?

Comment (eliperelman) [inline src/adb/logcat.js]: lol oops

File: src/routes/logs.js [hunk 3]
```diff
@@ -1,9 +1,26 @@
 let express = require('express');
+let fxos = require('../fxos');
+let moment = require('moment');
 
 let router = express.Router();
+const BUGGY_SDK_VERSION = 21;
```
Comment (gaye) [inline src/routes/logs.js]: Can you add a comment to explain what this is all about?

File: src/session.js [hunk 4]
```diff
@@ -5,7 +5,7 @@ let sessions = {};
 function hydrate(request, sessionId) {
   let session = sessions[sessionId];
 
-  Object.assign(request, session, { sessionId });
+  Object.assign(request, { session, sessionId });
```
Comment (gaye) [inline src/session.js]: Does this one mean `request.session = session;` and `request.sessionId = sessionId;`?

Comment (eliperelman) [inline src/session.js]: Yes, instead of randomly placing session variables on the request, organize them into a session object on the request.

Comment (eliperelman) [inline src/session.js]: It gives us a place to put future session variables.

File: src/fxos/getProperties.js (hunk 1 above)
Comment (eliperelman) [inline src/fxos/getProperties.js]: Haha, I understand. It's more of a documentation thing than anything. :)

File: src/session.js (hunk 4 above)
Comment (gaye) [inline src/session.js]: Sounds good!

File: src/routes/logs.js [hunk 5]
```diff
@@ -1,9 +1,40 @@
 let express = require('express');
+let fxos = require('../fxos');
+let moment = require('moment');
 
+/**
+ * There are two issues with `logcat -c` on Android LL
```
Comment (gaye) [inline src/routes/logs.js]: ++

**Pull request merged**